go.mod

```diff
@@ -24,14 +24,14 @@ require (
 	golang.org/x/crypto v0.0.0-20210616213533-5ff15b29337e
 	golang.org/x/sys v0.0.0-20210817190340-bfb29a6856f2
 	google.golang.org/grpc v1.40.0
-	k8s.io/api v0.22.1
-	k8s.io/apimachinery v0.22.1
+	k8s.io/api v0.22.2
+	k8s.io/apimachinery v0.22.2
 	k8s.io/client-go v12.0.0+incompatible
-	k8s.io/cloud-provider v1.22.1
+	k8s.io/cloud-provider v0.22.2
```

Comment:
Fixed this too https://pkg.go.dev/k8s.io/cloud-provider 
there's no v1.22.1 package